src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimController.java

```diff
@@ -105,8 +105,19 @@ public Claim save(
         return claimService.saveClaim(submitterId, claimData, authorisation, features);
     }
 
+    @PostMapping(value = "/{submitterId}/create-legal-rep-claim", consumes = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    @ApiOperation("Creates a new legal rep claim")
+    public Claim saveLegalRepresentedClaim(
+        @Valid @NotNull @RequestBody ClaimData claimData,
+        @PathVariable("submitterId") String submitterId,
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+        @RequestHeader(value = "Features", required = false) List<String> features
```

Comment:
there's a conversation about removing features which is marked as resolved, but features is still here